app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,60 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :firstname %><br />
+    <%= f.text_field :firstname, autofocus: true, required: true %>
+  </div>
+
+  <div class="field">
+    <%= f.label :lastname %><br />
+    <%= f.text_field :lastname, autofocus: true %>
+  </div>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.text_field :email, autofocus: true, required: true %>
+  </div>
+
+  <div class="field">
+    <%= f.label :country %><br />
+    <%= f.country_select :country %>
+  </div>
+
+  <div class="field">
+    <%= f.label :contact %><br />
+    <%= f.telephone_field :contact, autofocus: true, required: true, pattern: "[0-9]{3}-[0-9]{2}-[0-9]{3}", placeholder: "123-45-678" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %>
+    <em> (7 characters minimum) </em>s
+  </br>
+    <%= f.password_field :password, required:true, minlength: User::MIN_PASSWORD_LENGTH, pattern: '?=.*[a-z])(?=.*[A-Z])(?=.*\W)' %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password_confirmation %><br />
+    <%= f.password_field :password_confirmation, required: true %>
+  </div>
+  </br>
+
+  <p> By voluntarily registering on this site, you consent to: </p>
+  <p> (i) the collection, use and disclosure by the XYZ Institutions of your personal information, including your personal health information for the purposes of: (a) institutional planning to assist with the creation (b) notification and/or delivery of other programs and services (c) statistical analysis necessary to further assist  (d) planning of or response to a natural disaster or other event deemed to be an emergency (collectively, the "Purposes"); and (ii) being contacted by the XYZ Institutions for reasons related to the Purposes. </p>
+  <p> For more information please review our <%= link_to 'Privacy Policy.', "http://localhost:3000/pages/privacy_policy" %> </p>
```

Comment:
Why are not we using here (<%= link_to 'Privacy Policy.', "http://localhost:3000/pages/privacy_policy" %>) the rails helper methods?